src/components/FormAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { FirestoreCollection } from 'react-firestore';
+
+function normalizeItem(item) {
+  var pattern = /\d[.,\/#!$%\^&\*;:{}=\-_`~()]\ix/;
+  // remove capitilization
+  item.replace(pattern, '');
+  return;
+  // remove numbers and special characters
+  // spaces
+  //return item normalized
+}
+
+///// MOVED TO AddItemForm.js
+function FormAlert(props) {
+  console.log(normalizeItem(props.userSubmit));
```

Comment:
Reminding myself as much as anyone else here . . . remove `console.log`'s and commented code. Why is this so hard to remember in the moment?

Reply:
This whole `FormAlert.js` is not being used lol. I deleted the file. This was moved up a level into `AddItemForm.js`